src/.pylintrc

```diff
@@ -183,6 +183,7 @@ good-names=i,
            k,
            ex,
            Run,
+           ra,
```

Comment:
is `dec` already accepted?

Reply:
`dec` is three letters, so it doesn't get a pylint warning.